torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -770,9 +780,18 @@ class MKLDNNSubgraphSlicer {
       // conversions. from initial testing including it speeds up models
       case aten::max_pool2d:
       case aten::max_pool3d:
+      case aten::adaptive_avg_pool2d:
         return true;
     }
 
+    if (n->kind() == aten::hardtanh) {
+      auto min_val = constant_as<double>(n->namedInput("min_val")).value();
```

Comment:
Can we just support it generally ? It should be pretty easy to add without the constraint that it is equivalent to relu6 ?

Reply:
Tbh, I was only adding it because in the recent versions of torchvision, they are implementing `relu6` with `aten::hardtanh`. We could add the support for `hartanh` as long as  `min_val <= 0 <= max_val` but I'm not sure we really need it, though.

Reply:
Yea, I understand we are working through optimizing torchvision which i think makes sense, but here it would be marginally more work to support the op generally so I think we should just do it. I'm fine landing though.

BTW, next set of vision models we should look through are https://github.com/rwightman/pytorch-image-models/ and verify speedups / add more op support there